### ISSUE TITLE
Add info on ShortCut keys being remapped on macOS to User Interface chapter

### ISSUE THREAD

**Issue by kevinhendricks** [Sigil-Ebook/sigil-user-guide#2]
This is a reminder to myself to add info about ShortCut keys being remapped by Qt just on MacOS to our Sigil User's manual.

Something like quoting the following from Qt.  Possibly in an aside (box).

```
Note: On macOS, references to "Ctrl", Qt::CTRL, Qt::Key_Control 
and Qt::ControlModifier correspond to the Command keys on the 
Macintosh keyboard, and references to "Meta", Qt::META, 
Qt::Key_Meta and Qt::MetaModifier correspond to the Control keys.
```
Then add something like:

```
Using this convention allows each platform's most common shortcut 
key sequences to work as expected.  For example,  Ctrl+C will work 
for Windows/Linux for copy in Sigil.  And via Qt's automatic remapping
of Ctrl to CMD on macOS means that CMD+C will work for copy 
just as macOS users would expect.
```

**Comment by mizuOda** [comment]
Hi Kevin, wondering if I can help with this??

I'm not sure if all the Qt references are necessary or helpful to the average Sigil user. (It just makes things more confusing for me.) I'm thinking something like the following would suffice? 

```
Note for Mac users: References to the "Ctrl" key in Sigil correspond to the Command key on the Macintosh keyboard, and references to the "Meta" key correspond to the Control key on Mac. 

This is due to Qt's automatic remapping, which allows each platform's most common shortcut key sequences to work as expected. For example,  Ctrl+C will work for Windows/Linux for copy in Sigil, and via Qt's automatic remapping of Ctrl to Cmd on macOS, Cmd+C will work for copy, just as macOS users would expect.
```
If this works for you, I would be happy to do a pull request.

**Comment by kevinhendricks** [comment]
Yes, please do and I'll merge it.

**Comment by mizuOda** [comment]
Okay, will do in the day next day or so

**Comment by kevinhendricks** [comment]
No rush!  Thanks.

**Comment by kevinhendricks** [comment]
Closing this as fixed in master by PR #8
